kusari/cmd/upload.go

```diff
@@ -28,9 +30,11 @@ func init() {
 	uploadcmd.Flags().BoolVar(&uploadOpenVex, "open-vex", false, "Indicate that this is an OpenVEX document (optional, only works with files)")
 	uploadcmd.Flags().StringVar(&uploadTag, "tag", "", "Tag value to set in the document wrapper upload meta (optional, e.g. govulncheck)")
 	uploadcmd.Flags().StringVar(&uploadSoftwareID, "software-id", "", "Kusari Platform Software ID value to set in the document wrapper upload meta (optional)")
-	uploadcmd.Flags().StringVar(&uploadSbomSubject, "sbom-subject", "", "Kusari Platform Software sbom subject substring value to set in the document wrapper upload meta (optional)")
+	uploadcmd.Flags().StringVar(&uploadSbomSubject, "sbom-subject", "", "Kusari Platform Software sbom subject substring value to set in the document wrapper upload meta (optional, for OpenVEX docs only)")
```

Comment:
This is a little confusing now because one would naively expect "sbom-subject" to set the SBOM subject for an SBOM that's being uploaded, but I don't know if we can do much about that except document around it.

Reply:
Yes... One option is to make it a double use param. For OpenVEX it could be the sbom subject, for sboms it could be the subject name. What do you think? Is that too confusing? We could then make the other param `sbom-subject-version` or even `sbom-version`.

Reply:
I think using the same flag name might be confusing.

Reply:
I don't like it, but I can live with it :-D